src/middlewared/middlewared/plugins/failover.py

```diff
@@ -1185,7 +1185,7 @@ async def service_remote(middleware, service, verb, options):
         # to not replicate to the remote controller
         return
 
-    ignore = ('system', 'nfs', 'netdata', 'truecommand', 'docker', 'libvirtd', 'libvirtd-guests')
+    ignore = ('system', 'nfs', 'netdata', 'truecommand', 'truesearch', 'docker', 'libvirtd', 'libvirtd-guests')
```

Comment:
Sets are more efficient at membership tests. This is trivial of course but a fun fact anyway.
```suggestion
    ignore = {'system', 'nfs', 'netdata', 'truecommand', 'truesearch', 'docker', 'libvirtd', 'libvirtd-guests'}
```